JobManagerExample/JobManagerExample/JobManagerExampleView.swift

```diff
@@ -0,0 +1,322 @@
+private struct JobView: View {
+    /// The job that this view shows data for.
+    var job: Job
+    /// The job's error in case of failure.
+    @State var error: Error?
+    /// The job's satus.
+    @State var status: Job.Status
+    /// The latest job message.
+    @State var message: JobMessage?
+    
+    /// Initializer that takes a job for which to show the data for.
+    init(job: Job) {
+        self.job = job
+        status = job.status
+    }
+    
+    /// A Boolean value indicating if the progress is showing.
+    var isShowingProgress: Bool {
+        job.status == .started || job.status == .paused
+    }
+    
+    /// A Boolean value indicating if the cancel button is showing.
+    var isShowingCancel: Bool {
+        job.status == .started || job.status == .paused
+    }
+    
+    /// A string for the job's type.
+    var jobType: String {
+        "\(type(of: job))"
+    }
+    
+    var body: some View {
+        VStack (alignment: .leading, spacing: 5) {
+            Text(jobType)
+            Text(status.displayText)
+                .font(.footnote)
+            if let error = error {
+                Text(error.localizedDescription)
+                    .font(.footnote)
+                    .foregroundColor(.secondary)
+                    .lineLimit(1)
+            } else {
+                if let message = message {
+                    Text(message.text)
+                        .font(.footnote)
+                        .foregroundColor(.secondary)
+                        .lineLimit(1)
+                }
+                if isShowingProgress {
+                    ProgressView(job.progress)
+                        .progressViewStyle(.linear)
+                        .padding(.vertical)
+                }
+                HStack {
+                    if status == .started {
+                        Button("Pause", action: job.pause)
+                    } else if status == .paused {
+                        Button("Resume", action: job.start)
+                    } else if status == .notStarted {
+                        Button("Start", action: job.start)
+                    }
+                    Spacer()
+                    if isShowingCancel {
+                        Button("Cancel") {
+                            Task { await job.cancel() }
+                        }
+                    }
+                }
+                .buttonStyle(.borderless)
+                .padding(.top, 2)
+            }
+        }
+        .padding()
+        .onReceive(job.$status) {
+            status = $0
+            if status == .failed || status == .succeeded {
+                notifyJobCompleted()
+            }
+        }
+        .onReceive(job.messages) {
+            Logger.jobManagerExample.debug("Job Message: \($0.text, privacy: .public)")
+            message = $0
+        }
+    }
+    
+    /// Posts a local notification that the job completed.
+    func notifyJobCompleted() {
```

Comment:
This works well!

![IMG_64BB0264B4B8-1](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/a7a3b7ce-d5e9-491b-a898-5b049b67de89)

One thing I did notice is that the badge doesn't get reset when the app is opened or the job is completed. Not a big issue but since we're showing how to set the badge should we also show how to remove it?

Reply:
I'm not aware how to fix that. I noticed that no matter how many times a job completes the badge always just says 1. I can dig into that.

Reply:
fixed